.github/workflows/ruby.yml

```diff
@@ -19,13 +19,13 @@ jobs:
     strategy:
       matrix:
         ruby: ["3.2", "3.3"]
-        rails_version: ["7.1.3.4", "7.2.0"]
-        blacklight_version: ["7.38.0"]
+        rails_version: ["7.1.5.1", "7.2.2.1"]
+        blacklight_version: ["7.38.0", "8.8.2"]
```

Comment:
This has changes that are not in main: https://github.com/sul-dlss/blacklight-hierarchy/blob/main/.github/workflows/ruby.yml#L23